renovate/default-config.json

```diff
@@ -2,26 +2,12 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "config:base",
-    "schedule:monthly",
+    "schedule:weekdays",
```

Comment:
Does this mean we might have several prs a day updating the versions? If so I think I'd prefer to keep it to monthly, but what do others think?

Reply:
`pre-commit` is going via branch, as `github-actions`. I have no idea how often other things update. But I would be hesitant to go as far as `monthly`, `weekly` would be fine.

For testing purposes, I'd like to get this in, but we can increase it if we feel it's too frequent.

Don't forget, the child repo can overwrite this schedule in their `.renovaterc.json`!